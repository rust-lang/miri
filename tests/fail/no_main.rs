//@error-pattern: miri can only run programs that have a main function
#![no_main]
