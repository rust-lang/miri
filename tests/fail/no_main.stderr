error: miri can only run programs that have a main function

error: aborting due to previous error

